templates/basic-code.yml

```
@@ -0,0 +1,20 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    provider: codeready-toolchain
+      name: basic-code
+objects:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    annotations:
+      openshift.io/description: ${USERNAME}-code
+      openshift.io/display-name: ${USERNAME}-code
+      openshift.io/requester: ${USERNAME}
+    labels:
+      provider: codeready-toolchain
+          name: ${USERNAME}
```

Is this formatting correct?
```suggestion
    name: ${USERNAME}
```

---

See my suggestion below.